diff --git a/node_modules/@react-native-community/netinfo/lib/commonjs/index.js b/node_modules/@react-native-community/netinfo/lib/commonjs/index.js
index 3509cc2..936b2c9 100644
--- a/node_modules/@react-native-community/netinfo/lib/commonjs/index.js
+++ b/node_modules/@react-native-community/netinfo/lib/commonjs/index.js
@@ -219,3 +219,4 @@ var _default = exports.default = {
   useNetInfoInstance
 };
 //# sourceMappingURL=index.js.map
+// patch for metro resolution
\ No newline at end of file
